### DISCUSSION TITLE
About the performance compare with original faceapi.js

### DISCUSSION THREAD

**Discussion by fainwza007** [vladmandic/face-api#168] [Q&A] [answered]
I want to ask you about the performance of your version and original version of faceapi.js
if i want to make a face attendance system is it good for use?

**Comment by vladmandic** [comment, marked as answer]
definitely faster since it uses much newer tensorflow/js engine, but for the same reason it does use a bit more memory since newer tfjs is larger. otherwise, its still the same library.

**Reply by fainwza007** [reply]
Thank you so much.

**Comment by fainwza007** [comment]
I have a question sir when I use a webcam to detect faces how to prevent real/fake faces?
